module-i18n.php

```diff
@@ -9,8 +9,8 @@
 	_x( 'Creates WebP versions for new JPEG image uploads if supported by the server.', 'module description', 'performance-lab' ),
 	_x( 'Enqueued Assets Health Check', 'module name', 'performance-lab' ),
 	_x( 'Adds a CSS and JS resource check in Site Health status.', 'module description', 'performance-lab' ),
-	_x( 'oEmbed Optimizer', 'module name', 'performance-lab' ),
-	_x( 'Optimize oEmbed loading with improved lazy loading.', 'module description', 'performance-lab' ),
+	_x( 'Optimize Embeds', 'module name', 'performance-lab' ),
+	_x( 'Optimize Embed loading with lazy frames and scripts.', 'module description', 'performance-lab' ),
```

Comment:
I'm also getting away from the "oEmbed" terminology here since most users think of these as "Embeds" - `oEmbed` is an implementation detail.

Reply:
Good call